perceval/rendering/canvas/latex_canvas.py

```diff
@@ -0,0 +1,216 @@
+# MIT License
+#
+# Copyright (c) 2022 Quandela
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# As a special exception, the copyright holders of exqalibur library give you
+# permission to combine exqalibur with code included in the standard release of
+# Perceval under the MIT license (or modified versions of such code). You may
+# copy and distribute such a combined system following the terms of the MIT
+# license for both exqalibur and Perceval. This exception for the usage of
+# exqalibur is limited to the python bindings used by Perceval.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from .canvas import Canvas
+
+tikz_implemented_colors = {
+    "red": True,
+    "green": True,
+    "blue": True,
+    "cyan": True,
+    "magenta": True,
+    "yellow": True,
+    "black": True,
+    "gray": True,
+    "white": True,
+    "darkgray": True,
+    "lightgray": True,
+    "brown": True,
+    "lime": True,
+    "olive": True,
+    "orange": True,
+    "pink": True,
+    "purple": True,
+    "teal": True,
+    "violet": True,
+    "darkred": "\\definecolor{darkred}{rgb}{0.55, 0.0, 0.0}",
+    "thistle": "\\definecolor{thistle}{rgb}{0.85, 0.75, 0.85}",
+}
```

Comment:
There are colors we use that are missing from this dictionary.
For instance, "lightpink" and "lightblue" are missing (maybe others).

You can check this out with the following code:
```python
import perceval as pcvl

cnot = pcvl.catalog["postprocessed cnot"].as_processor().build()
pcvl.pdisplay(cnot, output_format=pcvl.Format.LATEX, recursive=False)
pcvl.pdisplay(cnot, output_format=pcvl.Format.LATEX, recursive=True)
```